README.md

```diff
@@ -45,31 +56,34 @@ Generate autotools scripts:
 ./autogen.sh
 ```
 
-To compile LINKS run:
+To install LINKS:
 
 ```
-./configure && make
+./configure && make install
 ```
-* Do not run `install` command to copy required executables to your path. Add the `bin` folder to your path as described in the next section. Support for `install` command is under development.
-
-* *These steps worked on a CentOS 7 system with 128 CPU Intel(R) Xeon(R) CPU E7-8867 v3 @ 2.50GHz
-
-### Running LINKS v2.0.0
--------------------------------
+To install LINKS in a specific directory:
 
-Add `bin` folder to your path when you are in the top level directory of the source tree:
 ```
-export PATH=$(pwd)/bin:$PATH
+./configure --prefix=/LINKS/PATH && make install
 ```
 
-Users can run `LINKS` (shell script in bin folder) to run the whole pipeline smoothly with LINKS parameters (such as -d xx -t yy) as pre 2.0.0 versions.
+*These steps worked on a CentOS 7 system with 128 CPU Intel(R) Xeon(R) CPU E7-8867 v3 @ 2.50GHz
+
+### Running LINKS v2.0.1
+-------------------------------
+
+Users must have all executable files(`LINKS`,`LINKS-make`,`LINKS_CPP`,`LINKS.pl`) accesible in PATH directories.
 
-To run LINKS with default parameters with `bin/LINKS`, assuming `bin` folder is in path:
+To run LINKS with default parameters:
 ```
 LINKS -f NA1281_draft.fa -s NA1281_reads.fof
 ```
 
-* *LINKS v2.0.0 is implemented in C++ and perl. The C++ executable is `src/LINKS_CPP` and PERL executable is `src/LINKS.pl`. To run the full LINKS pipeline smoothly you can run `LINKS-make`. However `LINKS-make` is a Makefile and has different argument value assignment (such as d=xx t=yy instead of -d xx -t yy).
+* *LINKS v2.0.0+ is implemented in C++ and perl. The C++ executable is `src/LINKS_CPP` and PERL executable is `src/LINKS.pl`. To run the full LINKS pipeline smoothly you can run `LINKS-make`. However `LINKS-make` is a Makefile and has different argument value assignment (such as d=xx t=yy instead of -d xx -t yy).
```

Comment:
You can run the full pipeline using `LINKS`, which calls the Makefile, right? I think this should be clarified here - the fact that there is a Makefile under the hood shouldn't matter if the user can still use `LINKS` and specify parameters as before.

Reply:
Answer to the first question, yes. And yes it definitely makes sense to clarify it here. I made the necessary change.